CONTRIBUTING.md

```diff
@@ -0,0 +1,507 @@
+# Welcome! Thank you for contributing to Akka!
+
```

Comment:
is this the file from akka/akka?

Reply:
Yes.

Reply:
I took the latest version including the scalafmt stuff (change to "Most Akka projects use scalafmt...").